### ISSUE TITLE
Error in EfficientNet V2 MBConvBlock 

### ISSUE THREAD

**Issue by sebastian-sz** [keras-team/keras#17358]
**System information**.
- Have I written custom code (as opposed to using a stock example script provided in Keras): No.
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): N/A
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 2.11
- Python version: 3.9
- Bazel version (if compiling from source): N/A
- GPU model and memory: N/A
- Exact command to reproduce: N/A

**Describe the problem**.
There is an error in EfficientNetV2's MBConvBlock - the final conv layers are not applied if Squeeze and Excitation block is not applied. This is an error, found by @DavidLandup0 in [Keras-CV repo](https://github.com/keras-team/keras-cv/pull/1146#discussion_r1050338581).

"Luckily" for MBConvBlock the SE Block is **always** applied so no need to change the weights.

**Describe the current behavior**.
Currently, the model works by accident.

**Describe the expected behavior**.
The MBConvBlock should work correctly even without SE Block.

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**.

- Do you want to contribute a PR? (yes/no): yes I can.
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing):
Basically, we need to unindent lines 717-740 in [efficientnet_v2.py](https://github.com/keras-team/keras/blob/master/keras/applications/efficientnet_v2.py#L717)

**Standalone code to reproduce the issue**.
N/A

**Source code / logs**.
Current faulty [MBConvBlock implementation](https://github.com/keras-team/keras/blob/9831c04766a227ae2b8787c982e2b63d9b1fb295/keras/applications/efficientnet_v2.py#L717).
Current [FusedMBConvBlock implementation](https://github.com/keras-team/keras/blob/9831c04766a227ae2b8787c982e2b63d9b1fb295/keras/applications/efficientnet_v2.py#L745), **without this issue**.
David's [fixed implementation](https://github.com/keras-team/keras-cv/blob/a3ae2dade8f536e96ee51f22b7e0a938cf2e14b1/keras_cv/layers/mbconv.py).

Thanks again @DavidLandup0 for finding this issue.

**Comment by tilakrayal** [comment]
@sebastian-sz,
We would prefer a PR for the validation, Could you please feel free to submit a PR for the requested change to be made. Thank you!

**Comment by sebastian-sz** [comment]
@tilakrayal PR present at https://github.com/keras-team/keras/pull/17364

**Comment by tilakrayal** [comment]
The pr has been assigned for reviewing and once it is merged this issue will move to closed status.
